pkg/templater/funcs.go

```diff
@@ -45,19 +49,48 @@ func newCredentials(params *ControlledParams) (*Credentials, error) {
 		return nil, fmt.Errorf("create credentials for %+v: %w", params.User, ErrNotAuthorized)
 	}
 
+	var (
+		generatedCredsI interface{}
+		ok              bool
+		generatedCreds  map[string]*Credentials
+	)
```

Comment:
AFAIK, this variable declaration is not idiomatic to Go. Why do we need to declare `generatedCredsI` and `ok`?
Is it because of the conversion in line 63?

Reply:
Yes :-(

Reply:
Paging @nopcoder: is there a better way??!?